beacon-chain/rpc/validator/attester.go

```diff
@@ -190,11 +198,39 @@ func (vs *Server) SubscribeCommitteeSubnets(ctx context.Context, req *ethpb.Comm
 	if len(req.Slots) != len(req.CommitteeIds) && len(req.CommitteeIds) != len(req.IsAggregator) {
 		return nil, status.Error(codes.InvalidArgument, "request fields are not the same length")
 	}
+	if len(req.Slots) == 0 {
+		return &ptypes.Empty{}, nil
+	}
+
+	fetchValsLen := func(slot uint64) (uint64, error) {
+		wantedEpoch := helpers.SlotToEpoch(slot)
+		vals, err := vs.HeadFetcher.HeadValidatorsIndices(wantedEpoch)
+		if err != nil {
+			return 0, err
```

Comment:
Lets annotate the error with grpc status codez

Reply:
will add status codes to all errors here

Reply:
Workin on this